web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -273,7 +273,7 @@ const ToolTip: React.FC = ({ children }) => {
         onMouseLeave={handlePopoverClose}
         css={`
           position: absolute;
-          background: red;
+          background: ${p => p.theme.colors.error.main};
```

Comment:
this file for v12 looks slightly different which was why automatic backport failed